src/components/Figure/index.js

```diff
@@ -64,9 +65,48 @@ export default function Figure() {
           width: 50,
           height: 50,
           color: "black",
-          type: "Square",
+          type: "square",
         });
 
+        visualizer();
+      } else if (data === "scaleUp") {
+        scaleCount++;
+        const selectedFigure = objects.current[objectActualIndex.current];
```

Comment:
```suggestion
        scaleCount++;
        
        const selectedFigure = objects.current[objectActualIndex.current];
```

해당 코드는 다음과 같이 개행을 분리시켜주면 가독성이 올라갈것같습니다.